app/DynamicsAdapter/DynamicsAdapter.Web/Startup.cs

```diff
@@ -195,6 +197,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // registration of health endpoints see https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks
                 endpoints.MapHealthChecks("/health", new HealthCheckOptions
                 {
+                    AllowCachingResponses = false,
                     Predicate = _ => true,
```

Comment:
_ typo?

Reply:
No, it's a property name on the object (it's part of the healtcheck lib)